app/templates/package_category.hbs

```diff
@@ -7,12 +7,9 @@
           <i aria-hidden="true" class="fa fa-search"></i>
         {{/link-to}}
       </section>
-      <nav class="breadcrumbs">
-        {{#link-to 'browse' classNames="underline-link"}}{{t 'browse.all_goods'}}{{/link-to}}
-        {{#link-to 'package_category' model.id classNames="current"}}{{model.name}}{{/link-to}}
-        {{select-list class="category_select" content=selectCategories on-change=(action (mut selectedCategoryId)) prompt=(t 'browse.filter_results')}}
-        <i class="fa fa-caret-down category_select"></i>
-      </nav>
+      <div class="browse-item-navbar">
+        {{partial 'navbar'}}
```

Comment:
@abhinav098 do you think we can rename `navbar` to maybe something like `breadcrumbs` ?
I know I'll confuse `navbar` the bar at the top of the app, or the sidebar. You need "patrick-proof" names 😆

Reply:
sure, I'll rename it to `_breadcrumbs.hbs`